src/server/project.ts

```diff
@@ -606,6 +614,39 @@ namespace ts.server {
                 }
             }
 
+            if (hasChanges) {
+                const missingFilePaths = this.program.getMissingFilePaths() || emptyArray;

```

Comment:
Again, `getMissingFilePaths` should always be defined, so no need for `|| emptyArray`.